sdk/rust/src/lib.rs

```diff
@@ -8,8 +8,7 @@ pub mod outbound_http;
 /// Key/Value storage.
 pub mod key_value;
 
-/// Sqlite
-#[cfg(feature = "experimental")]
```

Comment:
The feature is still being turned on in some places. While this won't break anything, we might want to clean up those uses in the rest of the codebase.

Reply:
Thanks for the catch - fixed